build/configs/rtl8721csm/nettest/defconfig

```diff
@@ -229,7 +232,7 @@ CONFIG_AMEBAD_FLASH_PAGE_SIZE=256
 CONFIG_AMEBAD_FLASH_BLOCK_SIZE=4096
 CONFIG_AMEBAD_AUTOMOUNT=y
 CONFIG_AMEBAD_AUTOMOUNT_USERFS=y
-CONFIG_AMEBAD_AUTOMOUNT_USERFS_DEVNAME="/dev/smart0p7"
+CONFIG_AMEBAD_AUTOMOUNT_USERFS_DEVNAME="/dev/smart0p6"
```

Comment:
Please revert this and below.

Reply:
@sunghan-chang I reverted them as you said.